opsdroid/core.py

```diff
@@ -34,15 +34,19 @@
 
 
 class OpsDroid:
-    """Root object for opsdroid."""
+    """Root object for opsdroid.
+
+    """
 
     # pylint: disable=too-many-instance-attributes
     # All are reasonable in this case.
 
     instances = []
 
     def __init__(self, config=None):
-        """Start opsdroid."""
+        """Start opsdroid.
```

Comment:
Here you should add the Arg config to the docstring, you could do something like:

```
Args:
    config (dict or None):  Dictionary containing opsdroid configuration.